database/seeders/categoriesSeeder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Support\Str;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class categoriesSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        for($i=0;$i<5;$i++){
```

Comment:
```suggestion
        for ($i = 0; $i < 5; $i++) {
```
cách ra cho dễ nhìn chứ bn sao viết xít vào nhau thế :\ với lại nếu tạo dữ liệu ngẫu nhiên tại sao bn ko dùng Factory nhỉ?